test/port-finder-socket.test.js

```diff
@@ -46,22 +47,19 @@ function createServers (callback) {
 
       server.listen(sock, next);
       base++;
-      servers.push(server);
+      servers.push([server, socket]);
```

Comment:
also - why are we adding an array with `[server, socket`] here - vs what we did in the past?

Reply:
again, to run `fs.unlinkSync(socket);` on Windows, which we didn't do before, it seems. If different let me know thanks